src/components/WorkflowComponentList/WorkflowComponentList.tsx

```diff
@@ -8,31 +8,47 @@ import { useMutation } from "react-apollo-hooks";
 import GridRow from "@govuk-react/grid-row";
 import GridCol from "@govuk-react/grid-col";
 
-import { SettingsListItem, LinkButton, Title } from "../../theme";
-import { CreateWebsiteListingInput, GetWebsiteListingQuery } from "../../API";
+import { SettingsListItem, Title } from "../../theme";
 import { WebsiteListing } from "../../types";
 
+interface Application {
+    __typename: string;
+    id: string;
+    rank: number;
+    openApplication: string | null;
+    closeApplication: string | null;
+}
+
 interface Props {
     websiteListings: (WebsiteListing | null)[] | null;
+    applications: (Application | null)[] | null;
```

Comment:
Change these types to match something like `applications?: Array<Application| null>`. Making it optional means that `| undefined` is implied.

Reply:
Made these changes